jest.config.js

```diff
@@ -17,10 +17,10 @@ module.exports = {
   coveragePathIgnorePatterns: ["mocks", "test-utils.ts"],
   coverageThreshold: {
     global: {
-      statements: 70,
-      branches: 70,
-      functions: 70,
-      lines: 70,
+      statements: 65,
+      branches: 65,
+      functions: 65,
+      lines: 65,
```

Comment:
:joy: Out of context this looks funny as it reminds me of [this video](https://www.youtube.com/watch?v=rR4n-0KYeKQ).

All fun & jokes aside though, **there's a good reason for changing these codecov percentages to what they are & the skipping out of the tests.**

It's not mentioned on the PR's description, but this PR is made to address partly inaccurate CA information displayed on the FE UI. We know for a fact that if CA is being displayed, then it deserves to be displayed. The problem is the details about the CA since most CAs are being displayed on incorrect people _(has to do with person guids being swapped around between CA records for some reason)_.

So this PR as mentioned is only a temporary hotfix that will get rolled back whenever the data source corruption issue gets resolved. So all the commented tests & codecov configuration will get rolled back together with the UI changes.

Reply:
That video made me laugh, good one! I added your description to the PR as well.